fiftyone/utils/cvat.py

```diff
@@ -3236,6 +3315,16 @@ def task_url(self, task_id):
     def task_data_url(self, task_id):
         return "%s/data" % self.task_url(task_id)
 
+    def task_data_download_url(
+        self, task_id, frame_id, data_type="frame", quality="original"
+    ):
+        return "%s/data?type=%s&quality=%s&number=%d" % (
```

Comment:
Just checking: are `type` and `number` mandatory for videos here?

I guess you wouldn't have gone through the trouble of merging chunks if you could have avoided it, but if CVAT stores the original video that'd be much cleaner

Reply:
Right, `type` `quality` and `number` are mandatory and can only give you either the compressed or original video `chunks`, not the original video itself, so you need to provide the `number` of the chunk you want to download.